Gemfile.template

```diff
@@ -2,6 +2,18 @@
 # If you modify this file manually all comments and formatting will be lost.
 
 source "https://rubygems.org"
+
+if Dir.exist?("../logstash-extra")
+  Dir.open("../logstash-extra").each do |entry|
+    next if entry.start_with?('.')
+    path = "../logstash-extra/#{entry}"
+    next unless File.directory?(path)
+    name = entry =~ /^(.+)-logstash$/ ? $1 : entry 
+    puts "Loading development gem: '#{name}' from '#{path}'"
+    gem name, :path => path
+  end
+end
+
```

Comment:
I like the idea, but one thing that make me nervous, Installing x-pack is not side effect free and _will_ change some internals, like adding a new monitoring pipeline auto-magically, that could make running the unit test locally fails if the state is different that what we expect in the suite.